src/main/kotlin/uk/gov/justice/digital/hmpps/prisonercellallocationapi/model/dto/MoveToCellRequest.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.prisonercellallocationapi.model.dto
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class MoveToCellRequest(
+
+  @Schema(
+    description = "The establishment",
+    example = "MDI",
+  )
+  var agency: String,
+
+  @Schema(
+    description = "The unique reference number for the cell the person is being assigned to",
+    example = "25463",
+  )
+  var cellId: Long,
+
+  @Schema(
+    description = "The common name for the cell the person is being assigned to e.g",
+    example = "A-1-2-003",
+  )
+  var cellDescription: String,
+
+  @Schema(
+    description = "The unique reference number for the person being assigned the cell",
+    example = "G7685GO",
+  )
+  var prisonerId: String,
+
+  @Schema(
+    description = "The name of the person being assigned the cell",
+    example = "Aishard Odnairab",
+  )
+  var prisonerName: String,
```

Comment:
Isnt this redundant? It can be looked up dynamically in prisoner-search

Reply:
It can but as a first pass in order to not require the calling service(s) to perform this lookup after finding who is in a given cell (and not require this service to perform this lookup either) we're storing the person's name as provided when being told where they are in order that the same information can be returned when querying the service. We can at a later date have either this service, the calling service or a layer in between look up the current name from prisoner-search